src/main/java/com/softserve/repository/impl/GroupRepositoryImpl.java

```diff
@@ -64,7 +65,7 @@
     private static final String UPDATE_GROUP_OFFSET
             = "UPDATE Group g "
             + "SET g.sortingOrder = g.sortingOrder+1 "
-            + "WHERE g.sortingOrder >= :position";
+            + "WHERE g.sortingOrder >= :lowerPosition and g.sortingOrder < :upperPosition";
```

Comment:
why do you need this statement? and g.sortingOrder < :upperPosition"

Reply:
It is used for update, if we want to update sort order of some group by setting its order at the beginning, we do not need to change offset of the elements that come after group, so we change only in specific interval